### PR TITLE
Update example from makeRequest() to handle()

### PR DESCRIPTION
**Pull request by chrisdwheatley**
This PR updates a code example in the post https://web.dev/app-shell-ux-with-service-workers/ to use `handle` rather than `makeRequest`.

Based on https://github.com/GoogleChrome/workbox/pull/2123 `makeRequest` has been removed and `handle` is what should be used instead.

I followed https://philipwalton.com/articles/smaller-html-payloads-with-service-workers/ for an example of how `handle` should be used.

### CODE REVIEW

**Comment by samthor** [comment]
cc @philipwalton

File: src/site/content/en/reliable/app-shell-ux-with-service-workers/index.md
Comment (philipwalton) [inline src/site/content/en/reliable/app-shell-ux-with-service-workers/index.md:152]: In v6 these will also need to pass an event object, so we may as well include that now so it doesn't have to be changed again in a few weeks.

**Review by philipwalton — CHANGES_REQUESTED** [review]

**Comment by jpmedley** [comment]
@jeffposnick, please take a look.

File: src/site/content/en/reliable/app-shell-ux-with-service-workers/index.md
Comment (jeffposnick) [reply src/site/content/en/reliable/app-shell-ux-with-service-workers/index.md:152]: Good point. Though if we're thinking about this with an eye towards v6, maybe the example should be updated to use `PrecacheStrategy` directly, instead of having to configure `CacheFirst`?

Comment (philipwalton) [reply src/site/content/en/reliable/app-shell-ux-with-service-workers/index.md:152]: Yeah, that's a good idea, though not sure if that'll mean we have to wait to merge this until v6 comes out?

Comment (jeffposnick) [reply src/site/content/en/reliable/app-shell-ux-with-service-workers/index.md:152]: We can just merge this now (without requiring `event`) and then file an issue to remind us to update the post once v6 is fully released.

**Comment by jeffposnick** [comment]
Merging, with some additional changes that we'll make when Workbox v6 is fully released. Thanks for the contribution, @chrisdwheatley!

**Review by jpmedley — APPROVED** [review]

**Comment by jpmedley** [comment]
@philipwalton I can't merge until you sign off.

**Review by philipwalton — APPROVED** [review]

**Pull request merged**
